ecosystem/indexer/migrations/2022-05-19-174534_collections/up.sql

```diff
@@ -6,7 +6,7 @@ CREATE TABLE collections
     creator VARCHAR NOT NULL,
     name VARCHAR NOT NULL,
     description VARCHAR NOT NULL,
-    max_amount bigint,
+    max_amount VARCHAR NOT NULL,
```

Comment:
can we still define this with numberic types? Defining this as varchar would lose the capability of such queries 'where max_amount > xxx'.

https://www.postgresql.org/docs/current/datatype-numeric.html

Reply:
we use diesel to insert into the DB now. it throws error when inserting string value to max_amount field.